### DISCUSSION TITLE
How to fix device state after crash during OTA update?

### DISCUSSION THREAD

**Discussion by janpieper** [Koenkk/zigbee2mqtt#13514] [General]
I had a short power outage while I was updating some devices (OTA). Now those devices are still in "updating" state although no update is running.

![Screenshot from 2022-08-12 12-50-50](https://user-images.githubusercontent.com/426371/184340031-72a74d1d-2dc8-418e-9763-ed29f2146d92.png)

The `state.json` contains the following for one of these devices:

```javascript
{
  // ...
  "0xa4c138dc0f11229a": {
      "state": "ON",
      "indicator_mode": "on/off",
      "child_lock": "UNLOCK",
      "power_outage_memory": "off",
      "power": 109,
      "current": 0.76,
      "voltage": 231,
      "energy": 216.49,
      "update": {
          "state": "updating"                   // <----- Wrong state
      },
      "update_available": false
  },
  // ...
}
```

I already tried to edit the file and change the state to `available` or `idle`, but after restarting z2m, the state was `updating` again :man_shrugging: 

I found #12794 which seems to describe the same issue, but there is no fix as the state was magically fixed after some time.

Is there anything else I need to do? Or is there another way to fix the state? I am currently running the following z2m version:

![Screenshot from 2022-08-12 12-59-04](https://user-images.githubusercontent.com/426371/184341062-65c14224-496d-48b6-b9b5-52bd104ac500.png)

**Comment by ptheofan** [comment]
I have a similar issue. I was doing OTA of a TuYa socket. At some point zigbee2mqtt crashed (I renamed a different device and crashed due to database error, id already exists). After I restarted zigbee2mqtt the OTA is stuck to almost zero (was 41% before crash) and it will not go away (30mins have already passed).

It would be great if there was some button somewhere to reset the OTA state so we can click to restart it (and take responsibility for it of course)

Regardless, the device works fine.


**Update**
Clicking the **check all** button again appears to fix the issue and cleared the OTA state.
